datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/dashboard.py

```diff
@@ -0,0 +1,98 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import json
+import os
+import re
+
+import click
+import requests
+
+from ....utils import ensure_dir_exists, path_join, write_file
+from ...constants import get_root
+from ...utils import get_valid_integrations, load_manifest, write_manifest
+from ..console import CONTEXT_SETTINGS, abort
+
+BOARD_ID_PATTERN = r'datadoghq\.com/[^/]+/([^/]+)/.+'
+SCREEN_API = 'https://api.datadoghq.com/api/v1/screen/'
```

Comment:
This might be different depending on the org/environment.

We can probably put the api base url in config like api_key/app_key.

Reply:
Fixed!